tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py{27,34,35,36},pypy
+envlist = py{27,35,36,37,38},pypy
+skip_missing_interpreters = True
```

Comment:
I guess we can remove this as well?

Reply:
it won't hurt.

Reply:
So if an interpreter is removed in Travis, this will make our test suite to succeed silently? Is my understanding correct?

Reply:
Travis fails before that. https://travis-ci.com/datastax/python-driver/jobs/210891608

As a developer, it won't complain if you don't install all the supported interpreters though.